### ISSUE TITLE
Which param from TF2-yolo is the same as the ignore_thresh from darknet-yolo?

### ISSUE THREAD

**Issue by alextudx** [pythonlessons/TensorFlow-2.x-YOLOv3#122]
Hello everyone,

I have used both TF2 and Darknet to train my custom dataset. I have found some questions but could not google out the answer. 

By training on the darknet, we need to set the ignore_thresh before training, which will also effect the calculate of loss function during training. As a result, it will effect the final_weights. If we train the dataset with small ignore_thresh, we will get a bigger Detection_count by mAP testing. If we train the dataset with bigger ignore_thresh, we will get a smaller Detection_count by mAP testing. The question is: which param from your Framework has the same function as **ignore_thresh**? In my opinion, both score_threshold and IOU_threshold in this repository only effect the result of calculating the mAP.

Why I asked this question is, I used the same dataset, the same training and testing path betwee tf2 and darknet, but always got a better AP, mAP and accuracy by tf2, I want to find out the reason.

Thank you very much!
